.github/workflows/release.yaml

```diff
@@ -54,13 +58,32 @@ jobs:
           RELEASE: ${{ env.RELEASE_VERSION }}-ARM
         run: |
           cd opensearch-operator
-          pwd
-          ls
-          go version
-          go env
           aws ecr-public get-login-password --region us-east-1 | docker login --username AWS --password-stdin public.ecr.aws/opsterio
           docker build -f Dockerfile-ARM -t public.ecr.aws/opsterio/opensearch-operator:$RELEASE ./
           docker push public.ecr.aws/opsterio/opensearch-operator:$RELEASE
           docker tag public.ecr.aws/opsterio/opensearch-operator:$RELEASE public.ecr.aws/opsterio/opensearch-operator:latest-ARM
           docker push public.ecr.aws/opsterio/opensearch-operator:latest-ARM
-
+      - name: Copy-crd-to-chart-dir
+        run: |
+          cd opensearch-operator
+          cp and mv can be done in one call:
+          cp ./config/crd/bases/opensearch.opster.io_opensearchclusters.yaml ../charts/opensearch-operator/templates/opensearchclusters.opensearch.opster.io-crd.yaml
+          sed -i '' -e 's/^version.*/version: $RELEASE_VERSION/' charts/opensearch-operator/Chart.yaml
```

Comment:
This line belongs in the helm-release workflow, here appVersion and tag should be updated.
And did you try the sed command? Because when I try locally the `-i ''` is invalid and I must use only `-i`

Reply:
yeah, tested it, works great